src/pages/DataSubmissionForm.js

```diff
@@ -93,6 +98,20 @@ export const DataSubmissionForm = () => {
     formData[key] = value;
   };
 
+  const updateParentRenderState = ({ key, value }) => {
+    if (key === 'nihAnvilUse') {
+      const val = (isArray(value) && !isEmpty(value)) ? value[0] : value;
+      if (includes(val)([YES_NHGRI_YES_PHS_ID, YES_NHGRI_NO_PHS_ID, NO_NHGRI_YES_ANVIL])) {
+        setNihAdminRendered(true);
+        setNihDataManagementRendered(true);
+      }
+      if (includes(val)([NO_NHGRI_NO_ANVIL])) {
+        setNihAdminRendered(false);
+        setNihDataManagementRendered(false);
+      }
```

Comment:
We could condense this to an if-else. There are only 4 choices that can be made, and 3 of them always render true, one always renders false. However, this might be better for future extensibility in in that if there is another choice added later on with no render changes, then we don't have to do anything here. And if we do, it's clear what we need to do.